src/main/kotlin/me/zeroeightsix/kami/feature/module/Nuker.kt

```diff
@@ -120,7 +134,7 @@ object Nuker : Module() {
             state.block !is FluidBlock &&
             block.isWithinDistance(
                 player.pos,
-                hitRange
+                range
```

Comment:
`validate` is called quite often, so we don't want to recalculate `range` every time. Add a `range` parameter to `validate`, and let `validate`'s caller compute the range once. Preferably add the parameter to the callers as well, and only calculate `range` in the event listener.